wco-empty/manifest.webmanifest

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Empty Window Controls Overlay App",
+  "display": "standalone",
+  "display_modifiers": [
+    "window-controls-overlay"
+  ],
+  "display_override": [
+    "window-controls-overlay"
+  ],
+  "start_url": "./index.html",
+  "theme_color": "#10826b",
```

Comment:
IDK what color this is, but feel free to switch it up to make it unique (or leave it. your choice really)

Reply:
done.